sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/test/java/com/azure/monitor/opentelemetry/autoconfigure/AzureMonitorExportersEndToEndTest.java

```diff
@@ -119,6 +120,33 @@ public void testBuildLogExporter() throws Exception {
         validateLog(logTelemetryItem);
     }
 
+    @Test
+    public void testBuildLogExporterWithCustomEvent() throws Exception {
```

Comment:
`AzureMonitorExportersEndToEndTest`  is enable again on `main`.  Could you please rebase?